rke/k8s_rke_system_images.go

```diff
@@ -8682,6 +8770,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled out of band post v2.6.12
```

Comment:
So far the plan is to align this KDM release with 2.6.12 actually, so let's change the comment as below,
```suggestion
		// Enabled in v2.6.12
```